### PR TITLE
Location page styling part 1

### PR DESCRIPTION
**Pull request by mok232**
This PR changes the styling on the location page.

- Created an account for font awesome library so we can use the CDN instead of downloading all the files
- Changed favorites icon to a heart and removed borders around the button
- Positioned favorites and review buttons inline with the title
- Placed average rating inline with "Reviews"

### CODE REVIEW

File: location/templates/location.html [hunk 1]
```diff
@@ -40,14 +45,90 @@
 {% endblock%}
 
 {% block content %}
-<h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
+<div class="row">
+<div class="col col-sm-9">
+    <h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
```
Comment (ab7289) [inline location/templates/location.html]: nit: could we make this a little bit smaller? I think it'd fit better with the favorite/review button

Comment (mok232) [reply location/templates/location.html]: Changed to `<h3>`

Comment (ab7289) [reply location/templates/location.html]: looks great! if you just wouldn't mind rebasing your branch I'll take one final peek and approve.

Comment (mok232) [reply location/templates/location.html]: rebased

**Pull request merged**